utils/textmate/Syntaxes/carbon.tmLanguage.json

```
@@ -1,11 +1,13 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM.
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception.
```

```suggestion
// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```
We need our license headers to be formatted like this -- the first line wraps onto the second line, so shouldn't have a trailing period, and the third line is machine-readable, so can't have a trailing period.